bin/schema_generator.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+# TODO: Update schema mapping for validation
+# TODO: Handle overwriting glean schemas
+
+GCS_BUCKET="mozilla-generated-schemas"
+MPS_BRANCH="generated-schemas"
+
+# 0. Install dependencies
+
+cargo install --git https://github.com/acmiyaguchi/jsonschema-transpiler --branch 0.2 --force
```

Comment:
@acmiyaguchi is this still how the transpiler should be retrieved?

Reply:
Yes, though this will change very soon. I'm moving the repository to the mozilla org either tomorrow or next week, and making master the default branch. I'm also trying to generate binaries for cross platform linux use, but the infrastructure for that isn't set up (and installing cargo isn't too bad anyways).

Reply:
Great, keep us updated. The cross-platform binaries could be useful for MPS testing, but aren't required here.

Reply:
I moved the repo to the mozilla org and [published a package](https://crates.io/crates/jsonschema-transpiler). This can now be `cargo install jsonschema-transpiler`.

Reply:
Great news! Updating now.